### DISCUSSION TITLE
Cannot run some reach commands

### DISCUSSION THREAD

**Discussion by iBrainiac** [reach-sh/reach-lang#842] [Q&A] [answered]
I am unable to complete the setup for the reach because i cant run ./reach config since its displaying this error  
 ```reach: /app/tmp/cli/2022-04-22T08-10-59Z-v5E5/out.sh: openFile: does not exist (No such file or directory)``` 
AND ./reach update is displaying the error below
./reach update
 ```reach: /app/tmp/cli/2022-04-22T08-32-09Z-5EIw/out.sh: openFile: does not exist (No such file or directory) ```

i however can run 
 ```./reach ```
 ```./reach version ```

**Comment by jeapostrophe** [comment]
@mattaudesse Can you please help?

**Comment by mattaudesse** [comment, marked as answer]
@iBrainiac are you using Ubuntu?
https://docs.reach.sh/trouble/#ts-reachsudo

If not, what's your OS and how did you install Docker?

**Reply by iBrainiac** [reply]
i couldn't set up docker in my windows laptop

**Comment by iBrainiac** [comment]
Yes I'm using Ubuntu because docker couldn't be installed in my windows
laptop

**Reply by mattaudesse** [reply]
Please try https://docs.reach.sh/trouble/#p_13 and let us know if that fixes it.

**Reply by iBrainiac** [reply]
Thanks I managed to solve the issue in Ubuntu . The problem was thaat i installed docker using snap. Snap installation in Ubuntu creates permissions issues. I Uninstalled Docker and then re-installed using the apt command in the terminal and it worked.

**Comment by iBrainiac** [comment]
My windows machine doesn't have enough resources ( small ram and space ) so
I opted for my machine with Ubuntu instead
